lib/cli.ts

```diff
@@ -83,16 +99,31 @@ export interface INpmDtsArgs {
    */
   logLevel?: ELogLevel
 
+  /**
+   * Don't add an alias for the main NPM package file to the generated .d.ts source
+   */
+  noAlias?: boolean
+
   /**
    * Attempts to at least partially generate typings ignoring non-critical errors
    */
   force?: boolean
 
+  /**
+   * Basic tree-shaking on module level
+   */
+  shake?: 'off' | 'exportOnly' | 'allImports'
```

Comment:
Would it be possible to define and use TypeScript enum with values for off, exportOnly and allImports?